example/parameterized_test_example.dart

```diff
@@ -103,16 +94,37 @@ void main() {
     },
   );
 
+  // Test which is a async test
   parameterizedTest(
-    'Example with list of one type.',
+    'Example using a async test',
     [
-      1,
-      2,
-      3,
+      100,
+      200,
+      300,
     ],
-    p1((int value) {
-      final result = value < 4;
-      expect(result, true);
-    }),
+    (int value) async {
+      final millis = DateTime.now().millisecondsSinceEpoch;
+      await Future<void>.delayed(Duration(milliseconds: value));
```

Comment:
Is this actually time awaited? Can't we use a test time skip method? Or at least just don't include this as an example as it seems bad practice?

Reply:
Added note about bad pratice